wp-tevko-responsive-images.php

```diff
@@ -297,31 +297,130 @@ function tevkori_get_src_sizes( $id, $size = 'thumbnail' ) {
 }
 
 /**
- * Filter for extending image tag to include srcset attribute.
+ * Filters images in post content to add `srcset` and `sizes`.
  *
- * @see images_send_to_editor
- * @return string HTML for image.
+ * @since 3.0
+ *
+ * @param string $content The raw post content to be filtered.
+ * @return string Converted content with `srcset` and `sizes` added to images.
+ */
+function tevkori_filter_content_images( $content ) {
+	// Only match images in our uploads directory.
+	$uploads_dir = wp_upload_dir();
+	$path_to_upload_dir = $uploads_dir['baseurl'];
+
+	$content = preg_replace_callback( '|<img ([^>]+' . $path_to_upload_dir . '[^>]+)[\s?][\/?]>|i', '_tevkori_filter_content_images_callback', $content );
```

Comment:
@joemcgill 

I am not sure but when I look at this regex it appears to me that the first group will contain everything up to the first ```>``` that it finds. So if there is a space and slash before the closing tag, it will be included.
Shouldn't we remove ```[\s?][\/?]``` and then change line 418 from
```
$image_html = "<img " . $atts . " " . $srcset . " " . $sizes . " />";
```
to
```
$image_html = "<img $srcset . " " . $sizes . " " . $atts . ">";
```
?
The space and the slash will be part of ```$atts```.
Or am I wrong?